bin/i18n-update.sh

```diff
@@ -138,7 +138,7 @@ for (( index=0; index<${#PLUGINS[@]}; index+=3 )); do
   PLUGINS_WITH_ADAPTED_PATHS+=( "$PLUGIN_NAME" "$PROJECT_SLUG" "$ADJUSTED_PLUGIN_FOLDER" )
 done
 pushd gutenberg/packages/react-native-editor > /dev/null
-METRO_CONFIG="../../../metro.config.js" node bin/extract-used-strings "$USED_STRINGS_PATH" "${PLUGINS_WITH_ADAPTED_PATHS[@]}"
+METRO_CONFIG="../../../metro.config.js" node bin/extract-used-strings "../../../$USED_STRINGS_PATH" "${PLUGINS_WITH_ADAPTED_PATHS[@]}"
```

Comment:
@dcalhoun I noticed this change is incompatible with the npm command `i18n:update` ([reference](https://buildkite.com/automattic/gutenberg-mobile/builds/8809#018e2d4b-0900-4918-b6ab-9a7a268add59/584-1355)) because the value used in `USED_STRINGS_PATH` points to an absolute path:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/df168602b7965e471f8b4c4f23c3b2dd99c4bbf9/bin/i18n-update.sh#L87

I'm going to create another PR to address it.

Reply:
Solved in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6715.

Reply:
@fluiddot ah, thank you for addressing this oversight. 🙇🏻